modules/assim.sequential/R/sda.enkf.R

```diff
@@ -373,20 +388,17 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
     ## add process error
     X[1:N]  ~ dmnorm(X.mod[1:N], prec = q[1:N,1:N])
     
-    #agb linear
-    #y_star[1:N,1:N] <- X[1:N,1:N] #[choose]
-    
-    #f.comp non linear
-    #y_star <- X[1:9] / sum(X[1:9])
+    #observation operator
+    y_star[1:YN] <- y_star_create(X[1:YN])
```

Comment:
@araiho once concern here is generalization -- if we approve this PR then this observation operator becomes the default and will break analyses that rely on our previous (linear) observation operator. If we can't figure out how to generalize this to a flexible observation operator specification at the moment, then I think we should add back the 
```
   #agb linear
   #y_star[1:N,1:N] <- X[1:N,1:N] #[choose]
 ```
option so it's more clear this needs to be manually switched at the moment. 

@istfer @para2x be aware of this bit -- I might go ahead and merge so the SDA code is pulled together before the refactoring

Reply:
I added a flag that will only use the ratio for 'fcomp.' Let me know if that won't work